src/attr.rs

```diff
@@ -319,10 +319,10 @@ impl<'s> Iterator for AttributesIterMut<'s> {
     }
 }
 
-impl<'s> IntoIterator for &'s mut Attributes<'s> {
+impl<'s, 'a: 's> IntoIterator for &'a mut Attributes<'s> {
     type Item = (&'s mut &'s str, &'s mut AttributeValue<'s>);
```

Comment:
I think the `&'s mut AttributeValues<'s>` should be `&'a mut AttributeValues<'s>`. We want to mutably borrow the values only for the lifetime of the iterator (`'a`) and not for the lifetime of the source code (`'s`).

We also do not want `'a` to outlive `'s`, they should be independent so it can be changed to `<'s, 'a>`.